.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +26,65 @@
 <!-- How is this issue solved/fixed? What is the design? -->
 <!-- It's better if we elaborate -->
 
+### Server checklist
+<!-- A checklist for server code -->
+
+#### Catalog upgrade
+<!-- Is hdb_catalog version bumped? -->
+Is Hasura catalogue version bumped?
+- [ ] Yes
+  - [ ] Update docs with down SQL script
+- [ ] No
+
+#### Metadata
+<!-- Hasura metadata changes -->
+
+Any new metadata feature is added?
+
+- [ ] Yes;
+   Does `run_sql` auto manages the new metadata through schema diffing?
+  - [ ] Yes
+  - [ ] Not required
+- [ ] No
+
+#### Breaking changes
+
+- [ ] No Breaking changes
+
+1. Metadata API
+
+   Existing `query` types:-
+   - [ ] Modify `args` payload which is not backward compatible
+   - [ ] Behavioural change of the API
+   - [ ] Change in response `JSON` schema
+   - [ ] Change in error code
+   <!-- Add if anything not listed above -->
+
+2. GraphQL API
+
+   Schema Generation:-
+   <!-- Any changes in schema auto-generation logic -->
+   <!-- All GraphQL schema names are case sensitive -->
+   - [ ] Change in any `NamedType`
+   - [ ] Change in table field names
+     <!-- Add if anything not listed above -->
+
+   Schema Resolve:-
+   <!-- Any change in logic of resolving input request -->
+   - [ ] Change in treatment of `null` value for input fields
+     <!-- Add if anything not listed above -->
+
+3. Logging
+
+   - [ ] Log `JSON` schema
+   - [ ] Log `type` names
```

Comment:
What is our policy on the stability of the logging schema, anyway? Is it documented anywhere?

Reply:
Our logging is documented [here](https://docs.hasura.io/1.0/graphql/manual/deployment/logging.html#). Few users are using JSON logs to monitor the activity in production. Any change in the logging system may break their assumptions.